oxigen_mrp/models/product_template.py

```diff
@@ -27,4 +27,21 @@ def write(self, vals):
                         )
                         % mrp_type
                     )
+            if "mrp_type" in vals:
```

Comment:
Check if the value is different than the current one and 
`if "mrp_type" in vals and vals ['mrp_type'] != rec.mrp_type:`
And `mrp_type` only makes sense `if tracking == 'serial' `is there any api.constraint that guarantees this? If not it should be inside the `if` that checks the tracking
By the way, that `if` should be `if vals.get('tracking', rec.tracking) == "serial":`